pipeline-maven-database/src/test/java/org/jenkinsci/plugins/pipeline/maven/db/PipelineMavenPluginDaoAbstractTest.java

```diff
@@ -35,9 +35,10 @@
 
 import javax.sql.DataSource;
 
+import org.assertj.core.api.Assertions;
```

Comment:
there is a static import so why those changes ? and why this test in particular and not others ?

if you really want to change to non static, remove line 27 please

Reply:
interesting. thanks IDE refactoring when moving classes around :) 
fixed d694dc5